samples/resource_files/resource_files_example.R

```diff
@@ -77,18 +69,25 @@ registerDoAzureParallel(cluster)
 #
 storageAccountName <- "mystorageaccount"
 outputsContainer <- "nyc-taxi-graphs"
-createContainer(outputsContainer)
-outputSas <- createSasToken(permission = "w", sr = "c", outputsContainer)
+rAzureBatch::createContainer(outputsContainer)
+
+# permissions: r = read, w = write.
+outputSas <- rAzureBatch::createSasToken(permission = "rw", sr = "c", outputsContainer)
 
 # =======================================================
 # === Foreach with resourceFiles & writing to storage ===
 # =======================================================
 
-results <- foreach(i = 1:12, .packages = c("data.table", "ggplot2", "rAzureBatch")) %dopar% {
+results <- foreach(i = 1:12) %dopar% {
+
+  library(data.table)
```

Comment:
Use .packages instead

Reply:
@brnleehng why are you suggesting .packages in this case?

Reply:
My idea is that people will use .packages to ensure their packages are getting installed during job preparation time. If they don't use our sample cluster config, they might forget to install those packages first.